Source/Moya+Defaults.swift

```diff
@@ -3,8 +3,12 @@ import Alamofire
 /// These functions are default mappings to `MoyaProvider`'s properties: endpoints, requests, manager, etc.
 public extension MoyaProvider {
     public final class func DefaultEndpointMapping(target: Target) -> Endpoint<Target> {
-        let url = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString
-        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+        if let url = target.baseURL.URLByAppendingPathComponent(target.path) {
```

Comment:
@ashfurrow the test didn't pass
❌  /Users/distiller/Moya/Source/Moya+Defaults.swift:6:15: initializer for conditional binding must have Optional type, not 'NSURL'

        guard let url = target.baseURL.URLByAppendingPathComponent(target.path),
              ^

Do you known what?